content/docs/dvclive/get-started.md

```diff
@@ -22,6 +22,51 @@ live = Live()
 
 See [`Live()`](/doc/dvclive/api-reference/live) for details.
 
+### Log parameters
+
+<toggle>
+<tab title="Single parameter">
+
+```python
+live.log_param("num_classes", 10)
+live.log_param("metrics", ["accuracy", "mae"])
+live.log_param("training",
+    "epochs": 42,
+        "batch_size": 128,
+        "optimizer": {
+            "name": "adam",
+            "loss": "categorical_crossentropy",
+            "learning_rate": 1e-3,
+        },
+    }
+)
```

Comment:
All this detail is great for the API ref, but it's too much here. Let's leave it at the simple example to save space.